uaclient/cli.py

```diff
@@ -57,10 +57,52 @@
 
 
 class UAArgumentParser(argparse.ArgumentParser):
+    def __init__(
+        self,
+        base_desc: str = None,
+        non_beta_services_desc: "List[str]" = None,
+        beta_services_desc: "List[str]" = None,
```

Comment:
If we are adding new params to the subclassed __init__ method common practice for the specialization of subclasses generally seems to be adding the new subclass params to the end of the parent's params. This also avoids positional collisions/unexpected behavior if a caller unfamiliar with extended params provided positional args instead of named kwargs.